src/utils/spec/utils.spec.ts

```diff
@@ -166,7 +167,9 @@ describe('utils', () => {
       const relativePath = './my-path/xyz'
       const pathRelativeTo = '/home/abc'
 
-      const expectedAbsolutePath = '/home/abc/my-path/xyz'
+      const expectedAbsolutePath = isWindows()
```

Comment:
I suggest creating a variable `pathSep` and use it to join parts of the path.

Reply:
Updated